/**:
  ros__parameters:
    bev_origin: [250, 699]
    px_dist: 0.04
    car_width: 1.32
    car_length: 2.0
    car_back_to_drive_axle: 0.43
    car_back_to_steer_axle: 1.47
    steering_speed: 0.7
    range_thresh_max: 9.0
    range_thresh_min: 3.0
    min_gap_size: 2.0
    goal_angle: 0.0
    wall_follow_clearance_min: 1.3   
    wall_follow_clearance_max: 3.0
    min_speed: 3.4
    slow_speed: 3.3
    max_speed: 8.0
    one_bound_kp: 1.2
    small_angle_kp: 1.3
    large_angle_kp: 1.1
